Documentation/ceph-upgrade.md

```diff
@@ -50,12 +50,30 @@ With this upgrade guide, there are a few notes to consider:
 ## Patch Release Upgrades
 
 Unless otherwise noted due to extenuating requirements, upgrades from one patch release of Rook to
-another are as simple as updating the common resources and the image of the Rook operator. For example, when Rook v1.5.1 is
-released, the process of updating from v1.5.0 is as simple as running the following:
+another are as simple as updating the common resources and the image of the Rook operator. For
+example, when Rook v1.5.3 is released, the process of updating from v1.5.0 is as simple as running
+the following:
 
+First get the latest common resources manifests that contain the latest changes for Rook v1.5.
+```sh
+git clone --single-branch --branch v1.5.3 https://github.com/rook/rook.git
+cd rook/cluster/examples/kubernetes/ceph
+```
+
+If you have deployed the Rook Operator or the Ceph cluster into a different namespace than
+`rook-ceph`, update the common resource manifests to use your `ROOK_OPERATOR_NAMESPACE` and
+`ROOK_NAMESPACE` using `sed`.
+```sh
+sed -i.bak \
```

Comment:
Instead of duplicating this sed command, how about if we just link to the other section below. It's not a common use case to need to change the namespace, so I'd prefer to keep the common case simple for the patch upgrades for readability.